blog/2023-04-18-magazine-003.md

```diff
@@ -0,0 +1,325 @@
+
+Here are some of the topics we covered:
+
+- Defining API endpoints in Wing
+- Introducing the Redis resource
+- Running tests in the cloud
+- Q&A with Eric Johnson about serverless and Wing
+
+If you didn’t get a chance to catch it live, you can tune in here (ADD LINK) and check out our next shows on [Twitch](https://www.twitch.tv/winglangio).
+
+### The first Wing hackathon has started
+
+We're excited to announce that our first hackathon, known as the **wingathon**, has started!
+The schedule of the hackathon is as follows:
+
+- Thursday, April 13th - kick-off and apps presentation by the sub-teams
+- Tuesday, April 18th - first check in
+- Thursday, April 20th - second check in
+- Thursday, April 27th - apps presentations
+
+The goal of this hackathon is to push the boundaries of creativity and innovation by using **Wing and its existing surface area** to create a working system that is ready for actual use.
+You should also be able to use any AWS resource in your Wing application through `bring "@cdktf/provider-aws"` (However, we still don’t have a great way for you to define how this behaves in the simulator). We want to challenge our team to think outside the box and come up with unique solutions to real-world problems using Wing's capabilities.
+
+The hackathon is taking place around the world, and everyone is invited to participate at their capacity. And **don't say you didn't know** -- there will be prizes awarded for the coolest apps, so make sure you bring your A-game and show us what you got!
+
+For more details, join us in the **#wingathon-1** channel in our [Slack](https://t.winglang.io/slack). There's still time to join the fun!
+
+### Community support for AWS CDK target introduced
+
+Thanks to the efforts of [Marcio](link), we now have community support for the AWS CDK target in Wing! This means you run `wing compile --target=awscdk` and get a CDK application that you can deploy to AWS using CloudFormation. Here's an example of a Wing app that uses the AWS CDK target:
+
+```js
+bring cloud;
+bring "aws-cdk-lib" as aws;
+
+let queue = new cloud.Queue();
+let bucket = new cloud.Bucket();
+
+queue.add_consumer(inflight (message: str) => {
+  bucket.put("latest_message.txt", message);
+});
+
+let func = new cloud.Function(inflight () => {
+  queue.push("hello world!");
+});
+
+let table = new aws.aws_dynamodb.Table();
+```
+
+As you can see, several resources from our batteries-included SDK are available in the AWS CDK target. Whenever you use these resources, your app can be deployed **both** to AWS using CloudFormation or to the Wing simulator.
+
+Since we are compiling to AWS CDK, we are also able to use any constructs from `aws-cdk-lib` directly in Wing.
+Since these imported resources don't know anything about Wing's "inflights", IAM permissions for these resources can't be inferred automatically by the compiler, and they do anything in the Wing simulator.
```

Comment:
```suggestion
Since these imported resources don't know anything about Wing's "inflights", IAM permissions for these resources can't be inferred automatically by the compiler, and they can't do a thing in the Wing simulator.
```